HPXMLtoOpenStudio/resources/EPvalidator.rb

```diff
@@ -153,6 +153,7 @@ def self.run_validator(hpxml_doc)
         'NumberofBathrooms' => zero_or_one,
         'ConditionedFloorArea' => one,
         'ConditionedBuildingVolume | AverageCeilingHeight' => one_or_more,
+        'HasFlueOrChimney' => zero_or_one,
```

Comment:
Should be 'extension/HasFlueOrChimney', right? Or were you thinking of formally proposing this for the HPXML schema?

Reply:
Should be 'extension/HasFlueOrChimney'.